Assignment6-CRUD_FastAPI/config.py

```diff
@@ -0,0 +1,23 @@
+from dotenv import load_dotenv
+import os
+from pydantic import BaseModel
+
+load_dotenv()
+
+
+class Config(BaseModel):
```

Comment:
Take a look at this in terms of classes. Python has an __init__ function you can define where it's values are set upon instantiation.

https://www.geeksforgeeks.org/__init__-in-python/